tests/unit_tests/rng/service/skip_ahead.cpp

```diff
@@ -25,44 +25,44 @@ extern std::vector<cl::sycl::device*> devices;
 
 namespace {
 
-class Philox4x32x10SkipAheadTests : public ::testing::TestWithParam<cl::sycl::device*> {};
+// class Philox4x32x10SkipAheadTests : public ::testing::TestWithParam<cl::sycl::device*> {};
```

Comment:
Why are these tests commented?
Does curandSetGeneratorOffset() work in a different way than skip_ahead?
SkipAheadEx should be fine, as the test would be skipped in case of unimplemented

Reply:
Thanks for spotting this! These tests were commented earlier for testing. Fixed.

Reply:
Attaching the `ctest` log for closure.
[ctest_curand.LOG](https://github.com/oneapi-src/oneMKL/files/5989107/ctest_curand.LOG)

Note that since cuRAND doesn't accept initializer lists for seeds, `skip_ahead` tests do not run.